README.md

```diff
@@ -1,5 +1,5 @@
 # Open Event Frontend
-![Open Event Frontend](https://storage.googleapis.com/eventyay.com/assets/branding/frontend_branding.png)
+![Open Event Frontend](public/images/Frontend_Landing.png)
```

Comment:
IMO moving the image to docs folder will be better since it belongs to documentation area.

Reply:
Have a look now @pradeepgangwar